File: 0001-TwoSum/main.py
class Solution:
    def twoSum(self, nums: list[int], target: int) -> list[int]:
        i: int = 0

        while i < len(nums):
            j: int = i + 1

            while j < len(nums):
                if nums[i] + nums[j] == target:
                    return [i, j]

                j = j + 1
                continue

            i = i + 1
            continue

        return [-1, -1]

File: 0001-TwoSum/main.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */

var twoSum = function (nums, target) {
  i = 0;

  while (i < nums.length) {
    j = i + 1;

    while (j < nums.length) {
      if (nums.at(i) + nums.at(j) == target) return [i, j];

      j++;
      continue;
    }

    i++;
    continue;
  }
};
